samples/model-builder/deploy_model_sample.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from google.cloud import aiplatform
+from typing import Optional, Sequence, Tuple, Dict
+from google.cloud.aiplatform import explain
+
+#  [START aiplatform_sdk_deploy_model_sample]
+def deploy_model_sample(
+    project,
+    location,
+    model_name: str,
+    endpoint: Optional["Endpoint"] = None,
+    deployed_model_display_name: Optional[str] = None,
+    traffic_percentage: Optional[int] = 0,
+    traffic_split: Optional[Dict[str, int]] = None,
+    min_replica_count: Optional[int] = 1,
```

Comment:
Same comment on `min_replica_count` and `max_replica_count`

Reply:
Done.